driver-core/src/main/com/mongodb/internal/session/ServerSessionPool.java

```diff
@@ -153,7 +153,7 @@ private void endClosedSessions(final List<BsonDocument> identifiers) {
                     }
                     return Collections.emptyList();
                 }
-            }).getConnection();
+            }).getServer().getConnection();
```

Comment:
Since the non-tuple based selectServer method has been removed, we need to use the tuple-based one here, and that requires use of the tuple's `getServer` method to get the Server

Reply:
That makes sense.